AUTHORING_GUIDE.md

```diff
@@ -617,6 +673,19 @@ To run a specific test from a specific following:
 nox -s py-3.7 -- snippets_test.py:test_list_blobs
 ```
 
+#### `noxfile_config.py`
+
+The [`noxfile_config.py`](noxfile_config.py) allows for customization
+of some options:
+
+* Ignore specific Python versions.
+* Enforce type hints.
+* Specify a different Google Cloud Project.
+* Add additional environment variables. Also see [Environment Variables](#environment-variables).
+
```

Comment:
```suggestion
* Override the version of `pip` used by nox

```

You can leave this out if you want, but I realized I didn't add this when I made the change 😁